fleche/dune

```diff
@@ -1,4 +1,4 @@
 (library
  (name fleche)
  (public_name coq-lsp.fleche)
- (libraries coq))
+ (libraries coq camomile))
```

Comment:
We don't need to depend on camomile, actually what we need is so simple that we could maybe just vendor it, should be very few lines of code (actually Coq already has the required UTF-8 functions but they are not exporting them)